consumer-test/integrationTestController.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+function fail() {
+    local message="$1"
+    local returnCode=${2:-1}
+    echo "[ERROR] ${message}" >&2
+    exit "${returnCode}"
+}
+
+function notify() {
+
+    local state=${1}
+    local description=${2}
+    local hash=${3}
+
+    echo "[INFO] Notifying about state \"${state}\" for commit \"${hash}\"."
+
+    curl -X POST \
+        --fail \
+        --silent \
+        --output /dev/null \
+        --data "{\"state\": \"${state}\", \"target_url\": \"${TRAVIS_BUILD_WEB_URL}\", \"description\": \"${description}\", \"context\": \"integration-tests\"}" \
+        --user "${INTEGRATION_TEST_VOTING_USER}:${INTEGRATION_TEST_VOTING_TOKEN}" \
+    "https://api.github.com/repos/SAP/jenkins-library/statuses/${hash}" || fail "Cannot send notification. curl return code: $?"
+}
+
+function cleanup() {
```

Comment:
There is a heart-beat job emitting a message every 10 seonds in the log. In case there is no output for some period of time travis kills the build. This we prevent hereby. In case there is some trouble with stopping that heart-beat job we ensure that this job does not continu ...